### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by yassin-kammoun-sonarsource**
The numbers were updated according to Peach.

### CODE REVIEW

**Comment by yassin-kammoun-sonarsource** [comment]
The numbers were updated according to Peach.

File: README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ This SonarSource project is a [static code analyser](https://en.wikipedia.org/wi
 # Features
 
 - Advanced rules based on pattern matching and control flow analysis
-- ~280 rules for [JavaScript](https://rules.sonarsource.com/javascript) and [TypeScript](https://rules.sonarsource.com/typescript)
-- ~30 rules for [CSS](https://rules.sonarsource.com/css)
+- [340 JS rules](https://rules.sonarsource.com/javascript) and [340+ TS rules](https://rules.sonarsource.com/typescript)
+- [25 CSS rules](https://rules.sonarsource.com/css)
 - Compatible with ECMAScript 2015-2020
-- React JSX, Flow and Vue support for JavaScript and TypeScript
+- React JSX, Flow, Vue, and AWS YAML support for JavaScript and TypeScript
```
Comment (ilia-kebets-sonarsource) [inline README.md:15]:
```suggestion
- React JSX, Flow, Vue, and AWS lambda functions support for JavaScript and TypeScript
```

**Review by ilia-kebets-sonarsource — APPROVED** [review]
WDYT about the suggestion, otherwise LGTM!

**Comment by yassin-kammoun-sonarsource** [comment]
In case we decide to drop a rule in the end, let's postpone the merge for now.

**Pull request merged**